R/data_match.R

```diff
@@ -183,6 +183,15 @@ data_filter <- function(x, ...) {
 #' @export
 data_filter.data.frame <- function(x, ...) {
   out <- x
+
+  # convert tibble to data.frame
+  if (inherits(data, "tbl_df")) {
+    out <- as.data.frame(out, stringsAsFactors = FALSE)
+    tbl_input <- TRUE
+  } else {
+    tbl_input <- FALSE
+  }
```

Comment:
I'll review more properly later but does this also strip other attributes from the `data`? For instance if there's a `geometry` column added by `sf`

Reply:
(gitblame says the code is from you 😎 )